src/Fields/Blueprint.php

```diff
@@ -108,14 +108,24 @@ public function setInitialPath(string $path)
 
     public function initialPath()
     {
+        if ($this->isNamespaced()) {
+            return $this->path();
+        }
```

Comment:
Do you remember why this was needed? 

The initial path is supposed to just be where the file actually was.
e.g. If it was in the namespaced directory, it'll be that. If it was customized by a user, it'll be the vendor path.

Everything seems to work fine if you remove this condition.